pylearn2/train_extensions/best_params.py

```diff
@@ -113,6 +113,10 @@ class MonitorBasedSaveBest(TrainExtension):
     store_best_model : bool, optional
         Whether to store the best model in memory. If False (the default),
         save_path must be defined.
+    start_epoch : int, optional
+        After this specified epoch, the model will start to be saved. Setting
+        this value to a resonable value prevents the program from saving the
+        model too much times at the begining of training.
```

Comment:
Hi Mehdi and David,

I've made the change on the docstring accordingly. Sorry for the typos...

Zhouhan


On Mon, Mar 2, 2015 at 2:04 PM, Mehdi Mirza <notifications@github.com>
wrote:

> In pylearn2/train_extensions/best_params.py
> <https://github.com/lisa-lab/pylearn2/pull/1417#discussion_r25626863>:
>
> > @@ -113,6 +113,10 @@ class MonitorBasedSaveBest(TrainExtension):
> >      store_best_model : bool, optional
> >          Whether to store the best model in memory. If False (the default),
> >          save_path must be defined.
> > +    start_epoch : int, optional
> > +        After this specified epoch, the model will start to be saved. Setting
> > +        this value to a resonable value prevents the program from saving the
> > +        model too much times at the begining of training.
>
> this -> the
> resonable -> reasonable
> program -> library
> much -> many
> begining -> beginning
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lisa-lab/pylearn2/pull/1417/files#r25626863>.
>



-- 
LIN, Zhouhan
Department of Computer Science and Operational Research
University of Montreal

Phone: +01-514-553-6037
Email: lin.zhouhan@gmail.com
Homepage:http://zhouhanlin.webs.com/